lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -154,6 +154,12 @@ sub _get_ptero_lsf_parameters {
     $set_lsf_option->('queue', $attributes{lsfQueue});
     $set_lsf_option->('projectName', $attributes{lsfProject});
 
+    my $default_job_group = join('/',
```

Comment:
Couldn't either the user set it or it could be parsed by `parse_lsf_params` (though that looks like it's currently parsing `-g` into `group` instead of `jobGroup`--maybe that should change!)?

The config value could also be changed, and the job would land in the desired group, and in a subgroup thereof that the user would almost certainly be able to further limit if they desired.